controller/controller_test.go

```diff
@@ -845,22 +881,52 @@ func createIngressesFixture(host string, serviceName string, servicePort int, in
 		}
 	}
 
-	return []*v1beta1.Ingress{
-		{
-			ObjectMeta: v1.ObjectMeta{
-				Name:        ingressName,
-				Namespace:   ingressNamespace,
-				Annotations: annotations,
+	if path == "" && host != "" {
```

Comment:
you could call:
`ingress:= createIngressWithoutRules()`
and then just populate the Spec fields as needed to avoid some of the repetition

Reply:
made the change